packages/ui/client/auto-imports.d.ts

```diff
@@ -108,6 +109,7 @@ declare global {
   const refDefault: typeof import('@vueuse/core')['refDefault']
   const refThrottled: typeof import('@vueuse/core')['refThrottled']
   const refWithControl: typeof import('@vueuse/core')['refWithControl']
+  const registerInitializeListener: typeof import('./composables/codemirror')['registerInitializeListener']
```

Comment:
remove this, no idea why not being removed, every time I add some composable I need to remove it if not being used and removed